lib/theme/common/forge-common.js

```diff
@@ -26,6 +26,18 @@
                 navigateTo(url)
             })
     }
+    const interceptLogOutClick = (url) => {
+        document.querySelector('#usermenu-item-logout')
+            .addEventListener('click', (e) => {
+                e.preventDefault()
+                e.stopPropagation()
+                if (context.shouldEmitInsteadOfRedirect) {
```

Comment:
Where is this flag set?

Reply:
There's a two way communication between the ff app and the editor:

1. Default being set in `nr-launcher/lib/theme/common/forge-common.js:10`
2. nr-launcher informs parent that it's loaded succesfully `lib/theme/common/forge-common.js:61`
3. ff app handles loaded event `frontend/src/pages/instance/Editor/components/EditorWrapper.vue:77`
4. ff app emits `prevent-redirect` message towards iframe with `true` payload
5. nr-launcher handles `prevent-redirect`  message and sets `context.shouldEmitInsteadOfRedirect` to whatever is received in message payload in `lib/theme/common/forge-common.js:55`

All communication done via postMessages